### ISSUE TITLE
Release exe just doesnt start [BUG]

### ISSUE THREAD

**Issue by TheRealDadbeard** [srevinsaju/guiscrcpy#269]
When I try to launch any of the exe files I just get a cmd windows with some errors and then it closes. I launched to exe in a cmd line and this is the error that I get

```
C:\Users\TheRealDadBeard\Downloads\scrcpy-win64-v1.20>guiscrcpy-PyQt5-x64.exe
Traceback (most recent call last):
  File "entry.py", line 1, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 546, in exec_module
  File "guiscrcpy\cli.py", line 12, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 546, in exec_module
  File "guiscrcpy\lib\utils.py", line 31, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 546, in exec_module
  File "guiscrcpy\platform\platform.py", line 25, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 546, in exec_module
  File "guiscrcpy\platform\windows.py", line 23, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 546, in exec_module
  File "guiscrcpy\platform\windows_tools\tools.py", line 10, in <module>
ModuleNotFoundError: No module named 'win32com'
[7536] Failed to execute script 'entry' due to unhandled exception!

C:\Users\TheRealDadBeard\Downloads\scrcpy-win64-v1.20>guiscrcpy-PySide2-x64.exe
Traceback (most recent call last):
  File "entry.py", line 1, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 546, in exec_module
  File "guiscrcpy\cli.py", line 12, in <module>
  File "C:\Users\THEREA~1\AppData\Local\Temp\embedded.v7qweele.zip\shibokensupport\__feature__.py", line 142, in _import
  File "PyInstaller\loader\pyimod03_importers.py", line 546, in exec_module
  File "guiscrcpy\lib\utils.py", line 31, in <module>
  File "C:\Users\THEREA~1\AppData\Local\Temp\embedded.v7qweele.zip\shibokensupport\__feature__.py", line 142, in _import
  File "PyInstaller\loader\pyimod03_importers.py", line 546, in exec_module
  File "guiscrcpy\platform\platform.py", line 25, in <module>
  File "C:\Users\THEREA~1\AppData\Local\Temp\embedded.v7qweele.zip\shibokensupport\__feature__.py", line 142, in _import
  File "PyInstaller\loader\pyimod03_importers.py", line 546, in exec_module
  File "guiscrcpy\platform\windows.py", line 23, in <module>
  File "C:\Users\THEREA~1\AppData\Local\Temp\embedded.v7qweele.zip\shibokensupport\__feature__.py", line 142, in _import
  File "PyInstaller\loader\pyimod03_importers.py", line 546, in exec_module
  File "guiscrcpy\platform\windows_tools\tools.py", line 10, in <module>
  File "C:\Users\THEREA~1\AppData\Local\Temp\embedded.v7qweele.zip\shibokensupport\__feature__.py", line 142, in _import
ModuleNotFoundError: No module named 'win32com'
[9600] Failed to execute script 'entry' due to unhandled exception!
```

**Comment by TheRealDadbeard** [comment]
I can't even install it with py:

```
Python 3.10.0 (tags/v3.10.0:b494f59, Oct  4 2021, 19:00:18) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> pip3 install -U guiscrcpy
  File "<stdin>", line 1
    pip3 install -U guiscrcpy
                 ^^^^^^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
>>>
```

Why is this so hard for me?

**Comment by srevinsaju** [comment]
Just use this for the time being: https://github.com/srevinsaju/guiscrcpy/suites/4145705937/artifacts/106301959

This error will be fixed in the coming release next week, so until then, the above exe should work. Let me know if you have any issues even with this exe.

**Comment by srevinsaju** [comment]
The fix for this released v4.11.3
